markdownlint/docs/TOP003.md

```diff
@@ -0,0 +1,138 @@
+# `TOP003` - Default Section Content
+
+Tags: `content`
+
+Aliases: `default-section-content`
+
+This rule is triggered when a section does not have the expected default content or structure. The affected sections include:
+
+- Lesson Overview
+- Assignment
+- Knowledge Check
+- Additional Resources
+
+Below are some examples of errors and their solutions:
+
+## Contains Content Before or After List
+
+There should be no content before or after the unordered list, other than the default content.
+
+Incorrect (content after the list):
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- LO item
+
+Some additional content
+```
+
+Incorrect (content before the list):
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+Some additional content
+
+- LO item
+```
+
+Correct:
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- LO item
+```
+
+## Contains Incorrect Default Content
+
+Each section should contain the correct default content.
+
+Incorrect (wrong default content before the list):
+
+```markdown
+### Lesson Overview
+
+This section has the incorrect text following the heading.
+
+- LO item
+```
+
+Correct:
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- LO item
+```
+
+Incorrect (missing the list after the `Additional Resources` heading):
+
+```markdown
+### Additional Resources
+
+This section contains helpful links to related content. It is not required, so consider it supplemental.
+```
+
+Correct:
+
+```markdown
+### Additional Resources
+
+This section contains helpful links to related content. It is not required, so consider it supplemental.
+
+- AR item
+```
+
+Incorrect (missing the list for Lesson Overview):
+
+```markdown
+### Lesson Overview
+
+This section contains helpful links to related content. It is not required, so consider it supplemental.
+```
+
+Correct:
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- LO item
+```
+
+Incorrect (contains a nested list):
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- An item
+  - A nested item
+```
+
+Correct:
+
+```markdown
+### Lesson Overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- An item
+- Another item
+```
+
```

Comment:
Let's add a heading for the Assignment section not having the expected div wrapper.

Reply:
It seems we're missing a test for that.

Reply:
Can you open a followup issue for this and ping me in it?

Reply:
I've included the test in this [commit](https://github.com/TheOdinProject/curriculum/pull/27545/commits/ee3302aa214d192a05e0f74b755e2bd93174d780). Do we still open a new issue?

Reply:
Oop nevermind 😆